tutorials/matchmaker101/director/main.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log"
+	"math/rand"
+	"sync"
+	"time"
+
+	"google.golang.org/grpc"
+	"open-match.dev/open-match/pkg/pb"
+)
+
+// The Director in this tutorial continously polls Open Match for the Match
+// Profiles and makes random assignments for the Tickets in the returned matches.
+
+const (
+	// The endpoint for the Open Match Backend service.
+	omBackendEndpoint       = "om-backend.open-match.svc.cluster.local:50505"
+	// The Host and Port for the Match Function service endpoint.
+	functionHostName        = "mm101-tutorial-matchfunction.mm101-tutorial.svc.cluster.local"
+	functionPort      int32 = 50502
+)
+
+func main() {
+	// Connect to Open Match Backend.
+	conn, err := grpc.Dial(omBackendEndpoint, grpc.WithInsecure())
+	if err != nil {
+		log.Fatalf("Failed to connect to Open Match Backend, got %w", err)
+	}
+
+	defer conn.Close()
+	be := pb.NewBackendClient(conn)
+
+	// Generate the profiles to fetch matches for.
+	profiles := generateProfiles()
+	log.Printf("Fetching matches for %v profiles", len(profiles))
+
+	for {
```

Comment:
```
for range time.Tick(time.Second * 5) {
```
Is simpler than sleeping, leads to more consistent behavior, and is more idiomatic.

Reply:
Reflected this.